compiler/linter/Visitor/ExpressionChecker.ml

```diff
@@ -0,0 +1,138 @@
+open ProgramVisitor
```

Comment:
Can you please delay the introduction of the program visitors to a future PR? Let’s write a quick-and-dirty ad-hoc visitor for the sake of the initialization checker and have this PR be of a manageable size. Thanks.

Reply:
Done.